src/client/components/ProductsPageView/ProductsPageView.component.js

```diff
@@ -9,10 +9,10 @@ export default function ProductsPageView({ products, productsPerPage }) {
   const [sortedProducts, setSortedProducts] = useState(products);
 
   const addFavorites = () => {
-    console.log('added to favorites');
+    // console.log('added to favorites');
```

Comment:
Please remove your comments entirely before committing.

Since this is a function that is left unimplemented you can leave a todo comment and it will be easy to find for other students as vscode has features to summarize all todos:

```suggestion
    // TODO: Implement add to favorites button
```